.idea/ansible-developer-vm.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I guess the gitignore did not already exist for the .idea/ansible-developer-vm.iml. Can you remove the file please?